doc/classes/Image.xml

```diff
@@ -760,7 +760,39 @@
 		<constant name="FORMAT_ASTC_8x8_HDR" value="38" enum="Format">
 			Same format as [constant FORMAT_ASTC_8x8], but with the hint to let the GPU know it is used for HDR.
 		</constant>
-		<constant name="FORMAT_MAX" value="39" enum="Format">
+		<constant name="FORMAT_R16" value="39" enum="Format">
+			OpenGL texture format [code]GL_R16[/code] where there's one component, a 16-bit unsigned normalized integer value.
```

Comment:
```suggestion
			OpenGL texture format [code]GL_R16[/code] where there's one component, a 16-bit unsigned normalized integer value. Since the value is normalized, each component is clamped between [code]0.0[/code] and [code]1.0[/code] (inclusive).
```

Same for the other notes.